### ISSUE TITLE
Fix Auth token handling in CI to modified EBRAINS identificaton management

### ISSUE THREAD

**Issue by dickscheid** [FZJ-INM1-BDA/siibra-python#85]
Currently the CI unit tests are failing due to invalid EBRAINS KG tokens. This is likely a consequence of the EBRAINS KG migration to the new identity management service, which needs to be fixed in the CI configuration.

**Comment by skoehnen** [comment]
@dickscheid Sorry I don't have the permission to access the necessary setting in the github project to set the credentials.
Please update my role so that I can perform these changes.

**Comment by xgui3783** [comment]
I believe KG has switched from oidc v1 to oidc v2 in the last few days. (if one goes to https://kg-editor.humanbrainproject.eu/ , and tries to login, you will be greeted with iam.ebrains.eu )

@dickscheid can you also add me to the role that can edit secrets?

@skoehnen if you would like to fix it, my understanding is that instead of using client id, client secret and refresh token, we just use client id and client secret, and use client credential flow to get a valid access token now days.

**Comment by skoehnen** [comment]
@xgui3783 yes, I think this was the cause of this. According to what I heard in some meetings v1 is still running in some form, but there were some changes on Wednesday last week that caused problems.

But we need to switch to v2 anyway, so let's better do it now ;)

**Comment by skoehnen** [comment]
Also a good moment to update some environment names, since this script was initially written for the JuGEx backend.

- [ ] Rename variables to siibra (from jugged)

**Comment by dickscheid** [comment]
> @dickscheid can you also add me to the role that can edit secrets?
should be done!

**Comment by skoehnen** [comment]
**Quick update:** I was unable to register the OIDC client because my credentials keep getting rejected, I am not sure what caused this, so I started support ticket.

**Comment by xgui3783** [comment]
this should be fixed? can we close it now? ping @skoehnen
